vimrc

```diff
@@ -145,6 +145,9 @@ nnoremap <Leader>r :RunInInteractiveShell<Space>
 " Treat <li> and <p> tags like the block tags they are
 let g:html_indent_tags = 'li\|p'
 
+"" Set tags for fugitive
+:set tags^=./.git/tags;
```

Comment:
Do we need the `./`? Can it be `set tags=.git/tags`? Also, I don't think you want the `:` at the front of the line.

Reply:
You are right! I just copied/pasted the message suggestion without even thinking 🙃